crates/next-dev/benches/mod.rs

```diff
@@ -230,21 +182,32 @@ fn bench_hmr_internal(mut g: BenchmarkGroup<WallTime>, location: CodeLocation) {
                                         "Unable to evaluate JavaScript in the page for HMR check \
                                          flag",
                                     )?;
-                                Ok(guard)
+
+                                // TODO(alexkirsz) Turbopack takes a few ms to start listening on
+                                // HMR, and we don't send updates retroactively, so we need to wait
+                                // before starting to make changes.
+                                // This should not be required.
+                                tokio::time::sleep(Duration::from_millis(5000)).await;
```

Comment:
We need to figure out a way to fix this race condition. I can still hit it even with 5s timeout.